fossilize.cpp

```diff
@@ -8041,7 +8041,7 @@ static bool json_value(const VkGraphicsPipelineCreateInfo &pipe,
 		cb.AddMember("logicOpEnable", pipe.pColorBlendState->logicOpEnable, alloc);
 		Value blend_constants(kArrayType);
 		for (auto &c : pipe.pColorBlendState->blendConstants)
-			blend_constants.PushBack(c, alloc);
+			blend_constants.PushBack(dynamic_info.blend_constants ? 0 : c, alloc);
```

Comment:
`0.0f` is preferable. I assume the int will be auto-promoted to float, but it gives me pause.

Reply:
Yes, it's more correct that way.